03/app.js

```diff
@@ -0,0 +1,24 @@
+const btnList = document.querySelectorAll('button');
+let counter = 0;
+
+let changeText = function() {
+    counter++; // tutaj jest cos nie tak ale nie wiem i chyba nie dojde do tego, pozwala mi kliknąc 5x 
+    
+    if (counter>=3) {
+        this.removeEventListener('click', changeText);
+    }
```

Comment:
To rozwiązanie pozwoli kliknąć 3 razy w ten sam button, a chodziło o to, aby kliknąć tylko raz w każdy.
Tutaj można by było pominąć warunek i będzie ok

Reply:
no tak! przecież wykona kliknięcie na każdym elemencie z  "nodeList" zmieni klasę i usunie nasluchiwanie, counter w ogóle jest zbędny, teraz to widzę, dzięki